website/settings/defaults.py

```diff
@@ -164,7 +164,7 @@ def parent_dir(path):
 # Mailchimp
 MAILCHIMP_API_KEY = None
 MAILCHIMP_WEBHOOK_SECRET_KEY = 'CHANGEME'  # OSF secret key to ensure webhook is secure
-ENABLE_EMAIL_SUBSCRIPTIONS = True
+ENABLE_EMAIL_SUBSCRIPTIONS = False
```

Comment:
Shouldn't this be True by default, but overriden as False in tests/locally as required?

Reply:
Yes. I was having issues with that test and set these, and forgot to revert.